app/src/main-process/menu/build-default-menu.ts

```diff
@@ -165,6 +167,14 @@ export function buildDefaultMenu({
         accelerator: 'CmdOrCtrl+2',
         click: emit('show-history'),
       },
+      {
+        label: getStashedChangesLabel(isStashedChangesVisible),
+        id: 'toggle-stashed-changes',
+        accelerator: 'Alt+1',
```

Comment:
The more I think about this menu item, the more it feels like "Go to Summary" which is in the next section. 

"Go to Summary" has an accelerator `CTRL+G`, which means "Show stashed changes"  could have something like `CTRL+H` (I'd have to scan for any clashes with this, but you get the idea).

Reply:
Changed to <kbd>Ctrl+H</kbd>